sdk/communication/Azure.Communication.CallAutomation/src/CallConnection.cs

```diff
@@ -656,5 +656,113 @@ public virtual CallMedia GetCallMedia()
                 throw;
             }
         }
+
+        /// <summary>
+        /// Mute participant from the call.
+        /// Only Acs Users are currently supported.
+        /// </summary>
+        /// <param name="targetParticipant">Participant to mute.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="targetParticipant"/> is null. </exception>
+        /// <returns>A Response containing MuteParticipantsResponse.</returns>
+        public virtual Response<MuteParticipantsResponse> MuteParticipants(CommunicationIdentifier targetParticipant, CancellationToken cancellationToken = default)
+        {
+            var options = new MuteParticipantsOptions(new List<CommunicationIdentifier> { targetParticipant });
+
+            return MuteParticipants(options, cancellationToken);
+        }
+
+        /// <summary>
+        /// Mute participants from the call.
+        /// Only Acs Users are currently supported.
+        /// </summary>
+        /// <param name="options">Options for the MuteParticipant operation.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="options"/> is null. </exception>
+        /// <returns>A Response containing MuteParticipantsResponse. </returns>
+        public virtual Response<MuteParticipantsResponse> MuteParticipants(MuteParticipantsOptions options, CancellationToken cancellationToken = default)
```

Comment:
Seems like need to replace MuteParticipantsResponse with MuteParticipantsReesult